boom! testing hlist-0.1.0 against stable for beta
boom! running: cargo +stable build --frozen
boom! creating container for: cargo +stable build --frozen
boom! running `docker create -v /home/ec2-user/cargobomb/./work/local/test-source/beta/stable:/source:ro -v /home/ec2-user/cargobomb/./work/local/cargo-home:/cargo-home:ro -v /home/ec2-user/cargobomb/./work/local/rustup-home:/rustup-home:ro -v /home/ec2-user/cargobomb/./work/local/target-dirs/beta/stable:/target:rw -e USER_ID=500 -e CMD=cargo +stable build --frozen cargobomb`
blam! a89ab3af63299e3c17769351d4d324010663cec925a26927a891834934f3d62e
boom! running `docker start -a a89ab3af63299e3c17769351d4d324010663cec925a26927a891834934f3d62e`
kablam!    Compiling hlist v0.1.0 (file:///source)
kablam!     Finished debug [unoptimized + debuginfo] target(s) in 0.23 secs
boom! running `docker rm -f a89ab3af63299e3c17769351d4d324010663cec925a26927a891834934f3d62e`
blam! a89ab3af63299e3c17769351d4d324010663cec925a26927a891834934f3d62e
boom! running: cargo +stable test --frozen --no-run
boom! creating container for: cargo +stable test --frozen --no-run
boom! running `docker create -v /home/ec2-user/cargobomb/./work/local/test-source/beta/stable:/source:ro -v /home/ec2-user/cargobomb/./work/local/cargo-home:/cargo-home:ro -v /home/ec2-user/cargobomb/./work/local/rustup-home:/rustup-home:ro -v /home/ec2-user/cargobomb/./work/local/target-dirs/beta/stable:/target:rw -e USER_ID=500 -e CMD=cargo +stable test --frozen --no-run cargobomb`
blam! 9fba9b0370da0779f04d9527ecbc0729db8ba0abb87a2855c52ea1df59397e54
boom! running `docker start -a 9fba9b0370da0779f04d9527ecbc0729db8ba0abb87a2855c52ea1df59397e54`
kablam!    Compiling hlist v0.1.0 (file:///source)
kablam!     Finished debug [unoptimized + debuginfo] target(s) in 0.50 secs
boom! running `docker rm -f 9fba9b0370da0779f04d9527ecbc0729db8ba0abb87a2855c52ea1df59397e54`
blam! 9fba9b0370da0779f04d9527ecbc0729db8ba0abb87a2855c52ea1df59397e54
boom! running: cargo +stable test --frozen
boom! creating container for: cargo +stable test --frozen
boom! running `docker create -v /home/ec2-user/cargobomb/./work/local/test-source/beta/stable:/source:ro -v /home/ec2-user/cargobomb/./work/local/cargo-home:/cargo-home:ro -v /home/ec2-user/cargobomb/./work/local/rustup-home:/rustup-home:ro -v /home/ec2-user/cargobomb/./work/local/target-dirs/beta/stable:/target:rw -e USER_ID=500 -e CMD=cargo +stable test --frozen cargobomb`
blam! f486a194691222437d4e15bdfdb6b05b06b3c548d0e0985d341ac879ab3f837e
boom! running `docker start -a f486a194691222437d4e15bdfdb6b05b06b3c548d0e0985d341ac879ab3f837e`
kablam!     Finished debug [unoptimized + debuginfo] target(s) in 0.0 secs
kablam!      Running /target/debug/deps/hlist-93fbd47aafa7db22
blam! 
blam! running 3 tests
blam! test test_get_mut ... ok
blam! test test_index_as_type_parameter ... ok
blam! test test_get ... ok
blam! 
blam! test result: ok. 3 passed; 0 failed; 0 ignored; 0 measured
blam! 
kablam!    Doc-tests hlist
blam! 
blam! running 4 tests
blam! test Find::get_mut_0 ... FAILED
blam! test Find_1 ... FAILED
blam! test Find::get_0 ... FAILED
blam! test Find_0 ... ok
blam! 
blam! failures:
blam! 
blam! ---- Find::get_mut_0 stdout ----
blam! 	error: expected `;`, found `let`
blam!  --> <anon>:5:1
blam!   |
blam! 5 | let mut list = Nil.push(0i32).push(1i64);
blam!   | ^^^
blam! 
blam! error: expected one of `.`, `;`, `?`, or an operator, found `assert`
blam!  --> <anon>:8:1
blam!   |
blam! 8 | assert!(a == 5);
blam!   | ^^^^^^
blam! 
blam! error: aborting due to previous error(s)
blam! 
blam! thread 'Find::get_mut_0' panicked at 'Box<Any>', /buildslave/rust-buildbot/slave/stable-dist-rustc-linux/build/src/librustc/session/mod.rs:201
blam! note: Run with `RUST_BACKTRACE=1` for a backtrace.
blam! 
blam! ---- Find_1 stdout ----
blam! 	error[E0405]: trait `Find` is not in scope
blam!  --> <anon>:2:14
blam!   |
blam! 2 | fn foo<I, L: Find<i32, I>>(list: &L) -> i32 {
blam!   |              ^^^^^^^^^^^^ `Find` is not in scope
blam!   |
blam!   = help: no candidates by the name of `Find` found in your project; maybe you misspelled the name or forgot to import an external crate?
blam! 
blam! error[E0425]: unresolved name `Nil`
blam!  --> <anon>:5:12
blam!   |
blam! 5 | let list = Nil.push("foo").push(5i32).push("bar");
blam!   |            ^^^ unresolved name
blam! 
blam! error: cannot continue compilation due to previous error
blam! 
blam! thread 'Find_1' panicked at 'Box<Any>', /buildslave/rust-buildbot/slave/stable-dist-rustc-linux/build/src/librustc/session/mod.rs:201
blam! thread 'Find_1' panicked at 'couldn't compile the test', /buildslave/rust-buildbot/slave/stable-dist-rustc-linux/build/src/librustdoc/test.rs:261
blam! 
blam! ---- Find::get_0 stdout ----
blam! 	error: expected `;`, found `let`
blam!  --> <anon>:5:1
blam!   |
blam! 5 | let list = Nil.push(0i32).push(1i64);
blam!   | ^^^
blam! 
blam! error[E0425]: unresolved name `list`
blam!  --> <anon>:6:15
blam!   |
blam! 6 | let a: i64 = *list.get();
blam!   |               ^^^^ unresolved name
blam! 
blam! error: aborting due to previous error(s)
blam! 
blam! thread 'Find::get_0' panicked at 'Box<Any>', /buildslave/rust-buildbot/slave/stable-dist-rustc-linux/build/src/librustc/session/mod.rs:201
blam! 
blam! 
blam! failures:
blam!     Find::get_0
blam!     Find::get_mut_0
blam!     Find_1
blam! 
blam! test result: FAILED. 1 passed; 3 failed; 0 ignored; 0 measured
blam! 
kablam! error: test failed
boom! running `docker rm -f f486a194691222437d4e15bdfdb6b05b06b3c548d0e0985d341ac879ab3f837e`
blam! f486a194691222437d4e15bdfdb6b05b06b3c548d0e0985d341ac879ab3f837e
